test/csv/interface/test_read.rb

```diff
@@ -264,6 +264,14 @@ def test_new_nil
     end
   end
 
+  def test_foreach_undef_replace
+    rows = []
+    CSV.foreach(@input.path, col_sep: "\t", row_sep: "\r\n", undef: :replace, replace: "*").each do |row|
+      rows << row
+    end
+    assert_equal(@rows, rows)
+  end
```

Comment:
We need to new test data that fails without these new options.

Reply:
I will make a new test data.